util/new_project.sh

```diff
@@ -2,11 +2,19 @@
 # Script to make a new quantum project
 # Jack Humbert 2015
 
-if [ -z "$1"  -o  -z "$2" ]; then
-	echo "Usage:   $0 <keyboard_name> <keyboard_type>"
-	echo "Example: $0 gh60 avr"
-	echo "Example: $0 bfake ps2avrgb"
-	exit 1
+KEYBOARD=$1
+KEYBOARD_TYPE=$2
+
+if [ -z "$KEYBOARD" ]; then
+   # Usage
```

Comment:
Sorry if I wasn't clear here. I meant that you should print the usage there, was just a bit lazy and didn't copy and paste that part. :)